subsys/bluetooth/mesh/Kconfig

```diff
@@ -1235,6 +1235,14 @@ config BT_MESH_DFD_SRV_TARGETS_MAX
 	  This value defines the maximum number of Target nodes the Firmware
 	  Distribution Server can target simultaneously.
 
+config BT_MESH_DFD_SRV_OOB_UPLOAD
+	bool "Support for DFU image OOB upload"
+	help
+	  This enables support for OOB upload of firmware images for
+	  distribution. This makes several callbacks and use of the init
+	  macro BT_MESH_DFD_SRV_INIT_OOB mandatory, see documentation for
```

Comment:
Do we have this documented or this means API documentation? If not should we document it addintionaly (in `dfu.rst`)? If should lets create a separate task for this.

Reply:
I was thinking API documentation yes. The new ones tell you that they are mandatory to support OOB upload. If you don't think this is enough, feel free to add the separate task